packages/components/transcript-editor/index.module.css

```diff
@@ -56,15 +56,15 @@
     position: relative;
   }
 
-  .header {
+  /* .header {
```

Comment:
if we don't need it, chuck it, or if you want to keep the class name in there for later, easier to deal with `//` comments imo.